pkg/apis/kubeone/v1beta1/types.go

```diff
@@ -524,6 +524,8 @@ type RegistryConfiguration struct {
 	// in OverwriteRegistry as an insecure registry. This is also propagated
 	// to the worker nodes managed by machine-controller and/or KubeOne.
 	InsecureRegistry bool `json:"insecureRegistry,omitempty"`
+	// Mirrors sets registry-mirrors
+	Mirrors []string `json:"mirrors"`
```

Comment:
What about the control plane nodes? I see that we provide it to machine-controller, but I guess it should affect control plane nodes as well.

Reply:
Good point